feature/foryou/src/main/java/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreen.kt

```diff
@@ -198,31 +220,43 @@ fun ForYouScreen(
 
             newsFeed(
                 feedState = feedState,
-                // Avoid showing a second loading wheel if we already are for the interests
-                // selection
-                showLoadingUIIfLoading =
-                interestsSelectionState !is ForYouInterestsSelectionUiState.Loading,
                 onNewsResourcesCheckedChanged = onNewsResourcesCheckedChanged,
-                loadingContentDescription = R.string.for_you_loading
             )
 
             item(span = { GridItemSpan(maxLineSpan) }) {
                 Column {
                     Spacer(modifier = Modifier.height(8.dp))
+                    // Add space for the content to clear the "offline" snackbar.
+                    if (isOffline) Spacer(modifier = Modifier.height(48.dp))
                     Spacer(Modifier.windowInsetsBottomHeight(WindowInsets.safeDrawing))
                 }
             }
         }
+        if (
```

Comment:
Nit: Could be nice to have some version of `AnimatedVisibility` here to animate appearing/disappearing.